osu.Game/Overlays/Settings/Sections/Graphics/DetailSettings.cs

```diff
@@ -19,22 +20,22 @@ private void load(OsuConfigManager config)
             {
                 new SettingsCheckbox
                 {
-                    LabelText = "Storyboard / Video",
+                    LabelText = GraphicsSettingsStrings.StoryboardVideo,
                     Current = config.GetBindable<bool>(OsuSetting.ShowStoryboard)
```

Comment:
> I noticed a pattern where most of the localisation naming is the same as the `OsuSetting` ones. Should we just be consistent and copy those where applicable instead?

@Joehuu I thought about that, but some of `OsuSettings`'name are missing context and/or not match the label string. the commenting line is the example.

Reply:
Let's not mix and match any more. Keeping the name of the localised string in line with its contents sounds best.

Reply:
> Let's not mix and match any more. Keeping the name of the localised string in line with its contents sounds best.

make sence, but I have to ask what's the threshold for (future) strings' name that too long to match content? 30 chars, less or more?
long string example (at least for me), 30 chars:
```csharp
/// <summary>
/// "Prefer metadata in original language"
/// </summary>
public static LocalisableString PreferOriginalMetadataLanguage => new TranslatableString(getKey(@"prefer_original"), @"Prefer metadata in original language");
```
extremely long string example:
```csharp
/// <summary>
/// "Using unlimited frame limiter can lead to stutters, bad performance and overheating. It will not improve perceived latency. "2x refresh rate" is recommended."
/// </summary>
public static LocalisableString UnlimitedFramesNote => new TranslatableString(getKey(@"unlimited_frames_note"), @"Using unlimited frame limiter can lead to stutters, bad performance and overheating. It will not improve perceived latency. ""2x refresh rate"" is recommended.");
```

Reply:
If you're asking me for a hard threshold number I won't really give you one. It's sort of situational to me. The string should preserve the original meaning as much as possible, but some adjustments are a good idea (appending `Header` to header text makes sense, for instance). The edge cases will just have to be a part of code review I'm afraid.